applicationset/generators/git.go

```diff
@@ -121,7 +121,7 @@ func (g *GitGenerator) generateParamsForGitFiles(appSetGenerator *argoprojiov1al
 	for _, requestedPath := range appSetGenerator.Git.Files {
 		files, err := g.repos.GetFiles(context.TODO(), appSetGenerator.Git.RepoURL, appSetGenerator.Git.Revision, requestedPath.Path, noRevisionCache, verifyCommit)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("%w", err)
```

Comment:
Is this intentionally a no-op? If so, I think we should just stick with `return nil, err`

Reply:
Yes, that's right, because when I tested it, `err` actually returned `"error generating params from git: paths error"`
If I format with `fmt.errorf ("error getting files from Git repository: %w", err")` like the previous commit, it will actually return `"error generating params from git: error getting files from Git repository: paths error".`
Return to `return nil, err`?

Reply:
Yep, I think the old way is fine for this one.

Reply:
I changed it!